woocommerce/woocommerce-ios
# Read-only Refunds Mark II

Closes #1577 

This PR continues the work on the Order Details > Refunded cells and the Net amount cells. Negative currency amounts now display in the correct order and format, according to the user's preferences in Settings on the web.

## Changes
- Updates the CurrencyFormatter to take negative currency values into account
- Updates unit tests that were passing but should not have been
- Updates unit tests to not use a calculated expected result

### To test
0. Create some refunds on some existing orders, using the web interface.
1. On your website, go to WooCommerce > Settings > General and scroll down to the Currency Options.
2. Save changes.
3. Check out this branch.
4. Build and run.

### Examples to test
Below are some example settings you can try and their expected results.

| Currency Options  | Order Details  |
|---|---|
| <img width="350" alt="set-USD-properly" src="https://user-images.githubusercontent.com/1062444/70093600-620e7f00-15e6-11ea-940e-74c89cb09f20.png"> | <img src="https://user-images.githubusercontent.com/1062444/70185511-8da96c00-16af-11ea-88a5-8decf4c82930.png" width="300" /> |
<img width="675" alt="set-kr-properly" src="https://user-images.githubusercontent.com/1062444/70148720-506fba80-166c-11ea-9f34-5694aefa44a8.png"> | <img src="https://user-images.githubusercontent.com/1062444/70186199-fd6c2680-16b0-11ea-81b5-367c73c2269c.png" width="300" /> |
| <img width="673" alt="aed" src="https://user-images.githubusercontent.com/1062444/70263547-3d3c1800-175c-11ea-8c94-abc4bea3bb58.png"> | <img src="https://user-images.githubusercontent.com/1062444/70263559-45945300-175c-11ea-8305-a88451668a06.png" width="300" /> |

Proper formatting for Arabic negative currency amounts should look like this: currency symbol `فلس`, space, minus sign `-`, amount value in arabic numbers. 

This is an example of the correct formatting but without arabic numbers:
<img width="480" alt="Screen Shot 2019-12-03 at 10 13 40" src="https://user-images.githubusercontent.com/1062444/70263680-8ab88500-175c-11ea-82f4-468265de2f3c.png">

#### Known Issues
- Order List currency formatting for order total in each cell is wrong, when testing RTL languages. This will be fixed when order total values display the value instead of being abbreviated.

Update release notes:
